CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-# Change Log
+# Changelog
+
+The [GitHub Releases](https://github.com/kubeflow/kubeflow/releases) page covers newer versions of `kubeflow/kubeflow` components.
```

Comment:
Does it make sense to list which components that release page covers (e.g. Central Dashboard, Notebooks, etc.) ?

Reply:
It covers everything, so that is probably not that useful.

It's the overall "platform version" so by proxy includes everything.

Reply:
@thesuperzapper Are we planing to remove this changelog from `kubeflow/kubeflow` once we split the repos ?

Reply:
This changelog is only here for historical reasons, it only includes stuff from 0.5 and earlier.

We may as well leave it once we split the repo, and leave the top section which links to the more modern changelogs.

Reply:
>We may as well leave it once we split the repo, and leave the top section which links to the more modern changelogs.

Yeah, but in that case you have to constantly update this changelog which links to changelogs of every Kubeflow components after Kubeflow release.
Do we want to do it in the future ?

Reply:
We should 100% have a page under this section of the website which includes those links:

- https://www.kubeflow.org/docs/releases/

And just link to that page.

Reply:
Should we just use section in README for it to minimize number of files in `kubeflow/kubeflow` repo ?

Reply:
As I was saying, google loves CHANGELOG.md files, so even a largely empty file which just links to the website is good.